### PR TITLE
Merge Pontoon integration

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,6 +42,12 @@
         'testing': [
             'psycopg2>=2.6',
         ],
+        'pontoon': [
+            'polib==1.1.0',
+            'pygit2==0.28.2',
+            'gitpython==3.0.2',
+            'toml==0.10.0',
```
Comment (merwok) [inline setup.py]: Pinned requirements are going to cause issues for apps using this.  Could you use ranges instead?  (like `polib == 1.1.*` or `pygit2 ~= 0.28.2` — both forms allow an app to pin a newer micro version (1.1.1, 0.28.3), the second form specifies a minimum required micro)

Comment (kaedroho) [reply setup.py]: Will change this. Thanks for pointing it out!

Comment (kaedroho) [reply setup.py]: Fixed (squashed into original commit)

File: docs/pontoon.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# wagtail-localize-pontoon
```
Comment (kaedroho) [inline docs/pontoon.md]: This will be rewritten, merging it in so it could be used as a reference for that rewrite.

**Comment by kaedroho** [comment]
Going to leave this separate for now to make it easier to get my other changes merged. I've ported the changes made here back to the original project https://github.com/wagtail/wagtail-localize-pontoon.

**Pull request closed without merging**